src/material/radio/radio.ts

```diff
@@ -338,8 +338,7 @@ class MatRadioButtonBase {
   // Since the disabled property is manually defined for the MatRadioButton and isn't set up in
   // the mixin base class. To be able to use the tabindex mixin, a disabled property must be
   // defined to properly work.
-  disabled: boolean;
-
+  get disabled(): boolean { return false; }
```

Comment:
Would it be possible to make this an `abstract` getter for the purpose of satisfying the `mixinTabindex`? Asking because otherwise we'd have an unnecessary getter adding unnecessary size. Previously it was type-only.

Same for other base classes following the same pattern.

Reply:
Yeah that should work. I’ll give it a try later.

Reply:
I am trying to go with the abstract approach but I am getting error in mixins such as
```ts
Argument of type 'Constructor<CanColor> & Constructor<HasElementRef>' is not assignable to parameter of type 'Constructor<CanDisable>'.
  Property 'disabled' is missing in type 'CanColor & HasElementRef' but required in type 'CanDisable'.
```

I am not sure if I should proceed with this change what do you reckon?

Reply:
For later reference: This how it seems to be do-able: https://gist.github.com/devversion/eec8ff59effba2ec83d8d197602081be